<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counter</class>
 <widget class="QWidget" name="Counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counter</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pantalla">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="getPeriodo">
    <property name="text">
     <string>Obtener período</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="setPeriodo">
     <property name="text">
      <string>Cambio período</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button">
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="getTimeElapsed">
    <property name="text">
     <string>Tiempo ejecución</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
